examples/pong.qasm

```diff
@@ -4,12 +4,12 @@
  * 2 PPPMMMMMMPPP
```

Comment:
I think this example needs a lot of updating. I tried to add grammatically correct syntax, but I am not sure if what is currently there is correct in terms of the physics. Should be checked.